zulipterminal/core.py

```diff
@@ -117,6 +117,7 @@ def show_msg_info(self, msg: Any) -> None:
 
     def search_messages(self, text: str) -> None:
         # Search for a text in messages
+        self.model.index['search'].clear()
```

Comment:
Does the test need to be amended here? Shouldn't it need to be?

Reply:
Rather 'added'. We didn't test for `self.model.index` contents after call to search_messages, but we can do that now.